bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/base/DatFile.scala

```diff
@@ -107,15 +107,15 @@ object DatFile {
       var dispersion: Double = 0
       try {
         dispersion = tmp.toDouble
-        l.+=(Grating(name, resolvingPower, blaze, dispersion, null, resolution))
+        l +=(Grating(name, resolvingPower, blaze, dispersion, null, resolution))
       } catch {
         case e : NumberFormatException => {
           val dDis = arrays.apply(tmp)
-          l.+=(Grating(name, resolvingPower, blaze, -999999, dDis, resolution))
+          l +=(Grating(name, resolvingPower, blaze, -999999, dDis, resolution))
         }
         case e : Exception => {
-          //l.+=(Grating(name, resolvingPower, blaze, -999999, arrays.apply(tmp), resolution))
-          println("NOT ANALYZED THE EXCEPTION, PLEASE CONTACT WITH A PROGRAMMER. LINE 125 OF THE DatFile.scala")
+
+          Log.severe("NOT ANALYZED THE EXCEPTION, PLEASE CONTACT WITH A PROGRAMMER. LINE 125 OF THE DatFile.scala")
```

Comment:
This is written as though we expect the user to see it?  Should we add the exception itself to the log message?  If we do we don't have to mention the line number (which is now incorrect anyway). Can we realistically recover from whatever we expect to happen though?  I'd be inclined to let it filter up.